.bazelrc

```diff
@@ -4,7 +4,7 @@
 
 # Recommended when using copy_directory, see
 # https://github.com/aspect-build/bazel-lib/blob/main/docs/copy_directory.md
---host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
+startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
```

Comment:
filed https://github.com/bazelbuild/bazel/issues/15245 cuz I couldn't find any other tracking bug for the root cause here